src/modules/75-cf/strings/strings.en.yaml

```
@@ -2014,21 +2013,20 @@ onboarding:
       ReactDOM.render(<App />, document.querySelector('#react-root'))
 
       function App() {
+
+        const reactTarget = { name: 'ReactClientSDK', identifier: 'reactclientsdk' }
+        const sdkApiKey = {{apiKey}}
```

Need quotes, or this will bomb
```suggestion
        const sdkApiKey = '{{apiKey}}'
```

---

My bad. thanks have addressed